Dialog.d.ts

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+
+export interface DialogProps extends React.DOMAttributes {
```

Comment:
I had my doubts about this. Since dialog passed the `other` to it's root div, it can inherit those attributes. I'm not so sure. I'll remove it, worst case-scenario some one will complain and we'll put it back :laughing: